### PR TITLE
fix_: pass proper version to params

### PR DESCRIPTION
**Pull request by igor-sirotin**
1. Fixed version calculation in Makefile:
    - Pure tag when there's no changes on top: `v0.184.45`
    - Tag with number of commits and commit hash: `v0.184.45-4-gdc9046b35` (`g` stands for `git` here)
     - Also `-dirty` is appended when there're any local uncommited changes: `v0.184.45-4-gdc9046b35-dirty`
2. Deleted `VERSION` file

### CODE REVIEW

**Comment by igor-sirotin** [comment]
@jakubgs @apentori can you guys please take a look?

We don't use `VERSION` file anymore, we store the version in git tags.
So I would like to delete it from the repo. What would be the proper way to update CI for this?

image

**Review by Samyoul — APPROVED** [review]
This fix looks so simple and so straightforward that I'm suspicious of it.

**Comment by jakubgs** [comment]
@igor-sirotin the proper way is to modify usage of `utils.pkgFilename()`:
https://github.com/status-im/status-go/blob/551af54fdac75288f134593354e08ea7015bf050/_assets/ci/Jenkinsfile.linux#L41
https://github.com/status-im/status-go/blob/551af54fdac75288f134593354e08ea7015bf050/_assets/ci/Jenkinsfile.android#L41
https://github.com/status-im/status-go/blob/551af54fdac75288f134593354e08ea7015bf050/_assets/ci/Jenkinsfile.ios#L41

Which calls `utils.getVersion()` if `version` is not provided:
https://github.com/status-im/status-jenkins-lib/blob/901787fcdec2f7e8a1278fbb4aafcffd49a1fc29/vars/utils.groovy#L49-L55

**Review by alaibe — APPROVED** [review]

File: _assets/scripts/version.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+git describe --tags --dirty="-dirty"
```
Comment (Samyoul) [inline _assets/scripts/version.sh:2]: @igor-sirotin Does this work? On my local machine I get the following

```bash
% git describe --tags --dirty="-dirty"
v0.50.2-3489-gda70d6f1b
```

Comment (Samyoul) [reply _assets/scripts/version.sh:2]: Ah, I've resolved this issue, I hadn't fetched the tags

Comment (Samyoul) [reply _assets/scripts/version.sh:2]: Works as expected

`develop`

```bash
% git checkout develop   
Already on 'develop'
Your branch is up to date with 'origin/develop'.
% git describe --tags --dirty="-dirty"
v0.184.46-3-gab675a2c9
```

`release/0.182.x`
```
% git checkout release/0.182.x   
Already on 'release/0.182.x'
Your branch is up to date with 'origin/release/0.182.x'.
% git describe --tags --dirty="-dirty"
v0.182.38-54-g89662aa30
```

Comment (igor-sirotin) [reply _assets/scripts/version.sh:2]: @Samyoul Interesting, works fine for me on the same commit.
Maybe you don't have the tags fetched?

```shell
[19/08/24 2:42:43] ➜  status-go git:(develop) ✗ git describe --tags --dirty="-dirty"
v0.182.38-48-gda70d6f1b
```

**Comment by jakubgs** [comment]
Your PR comment doesn't reflect the changes tho.

**Comment by jakubgs** [comment]
Some style fixes:
- https://github.com/status-im/status-go/pull/5747

**Pull request merged**